pkgs/by-name/gu/guile-goblins/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+, guile
+, guile-fibers
+, guile-gcrypt
+, texinfo
+, pkg-config
+}:
+stdenv.mkDerivation rec {
+  pname = "guile-goblins";
+  version = "0.11.0";
+
+  src = fetchurl {
+    url =
+      "https://spritely.institute/files/releases/guile-goblins/guile-goblins-${version}.tar.gz";
```

```suggestion
    url = "https://spritely.institute/files/releases/guile-goblins/guile-goblins-${version}.tar.gz";
```

Still don't need that line break.